.travis.yml

```diff
@@ -39,7 +45,7 @@ before_script:
 script:
   - sleep 10 # give enough time to the webserver to start
   - configure_shotgun --env-script "source activate qp-shotgun" --server-cert $QIITA_SERVER_CERT
-  - nosetests --with-doctest --with-coverage
+  - nosetests --with-doctest --with-coverage -e humann2
```

Comment:
Cool, didn't know this was possible, I guess it makes sense. Could you pull from master and remove it?